test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test

```diff
@@ -0,0 +1,153 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PATCH /v2/entities/E1
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Intent to patch a non-existing entity - error, of course
+# 02. Create E1/attr1+attr2
+# 03. GET /v2/entities/E1
+# 04. PATCH /v2/entities/E1 attr1=4
+# 05. GET /v2/entities/E1 - see attr1=4
```

Comment:
Add step: GET /v2/entities/E1 - see attr2 in order to check that the PATCH in step 4 hasn't modified the other attribute.